src/documents/IoCTestingFramework.md

```diff
@@ -333,25 +601,34 @@ fluid.defaults("fluid.tests.asyncTestTree", {
 });
 ```
 
+### `markupFixture` property supporting fixtures written against markup in the host document
+
 This environment shows use of the optional `markupFixture` property on the `testEnvironment`. Since the IoC testing framework
 operates setup/teardown on the unit of overall `testEnvironment`s, we cannot (should not) make use of QUnit's standard
 markup setup/teardown operated on the hard-wired DOM node with id `qunit-fixture`, which is on the unit of individual
 test cases. The `markupFixture` property is to be used where the overall environment makes use DOM material where its
-markup is rendered, which naturally should be reset to its original value between runs of different `testEnvironment`s.
+markup is rendered, which should be reset to its original value between runs of different `testEnvironment`s.
 The `markupFixture` property holds any jQueryable value, designating the overall root node of this DOM material. After
 the `testEnvironment` has been torn down, the framework will reset the markup within this root to the contents it
 enjoyed before setup of the environment.
 
-Finally, we show the contents of the associated `TestCaseHolder`. In this case, the 1 test it defines holds a sequence
-member prescribing a sequence of 11 states for the component, which run a total of 7 jqUnit assertions. These show
-records of all of the 5 types defined above - the framework ensure the correct sequence of activities (including
-binding and unbinding of listeners registered in `binder` records) is operated, as well as showing feedback in the
+When run in the browser, the framework will show feedback in the
 QUnit UI relating to the sequence point reached by the system. This can be used to diagnose the last successfully
 reached sequence point in the case of a "hang" caused by an unexpectedly missing event in the sequence.
 
+The IoC Testing Framework can also be used to run test sequences in the node.js server environment - in this case
+the above features are not provided.
```

Comment:
> in this case the above features are not provided.

That's a bit ambiguous. Is it only the feedback about sequence point being reached that isn't in node? Also how would someone debug a similar issue in node?